dashboard/app/views/devise/registrations/_old_sign_up_form.html.haml

```diff
@@ -117,16 +117,17 @@
       %br
       #terms_of_service_version-block.itemblock
         .error_in_field.checkbox-error
-        %br
-        = f.label :terms_of_service_version do
+        = f.label :terms_of_service_version, {class: 'terms'} do
```

Comment:
note that labels are [inline elements that should only contain phrasing content](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#Technical_summary)

Rather than trying to manipulate this label with CSS to support including the kind of content we want, I would suggest extracting the content that should not be contained in a label out to a sibling element, and updating the label to only contain label-like content.

ie, instead of something like

```html
<label>
  <input class="terms-checkbox" type="checkbox" value="1" name="user[terms_of_service_version]" id="user_terms_of_service_version">
  <div class="teacher-options" style="display: none">
    <p>I agree to the <a href="//code.org/tos" target="_blank">Terms of Service</a> and <a href="//code.org/privacy" target="_blank">Privacy Policy</a>.</p>
  </div>
  <div class="student-options">
    <p>I agree to the <a href="//code.org/tos" target="_blank">Terms of Service</a> and <a href="//code.org/privacy" target="_blank">Privacy Policy</a>.</p>
    <p>If I am under 13 years of age, I confirm that I have my parent or legal guardian's permission to use the Code.org services.</p>
  </div>
</label>
```

do something like

```html
<div>
  <div class="teacher-options" style="display: none">
    <p>I agree to the <a href="//code.org/tos" target="_blank">Terms of Service</a> and <a href="//code.org/privacy" target="_blank">Privacy Policy</a>.</p>
  </div>
  <div class="student-options">
    <p>I agree to the <a href="//code.org/tos" target="_blank">Terms of Service</a> and <a href="//code.org/privacy" target="_blank">Privacy Policy</a>.</p>
    <p>If I am under 13 years of age, I confirm that I have my parent or legal guardian's permission to use the Code.org services.</p>
  </div>
  <label>
    <input class="terms-checkbox" type="checkbox" value="1" name="user[terms_of_service_version]" id="user_terms_of_service_version">
    I agree
  </label>
</div>
```

Reply:
Done. Please checkout the new screenshots to preview what it looks like.